dist/s_fixme

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+# Find FIXME comments related to the ticket potentially indicated by the branch name.
+# The script only fails if the branch name follows a specific syntax and FIXME comments are found,
+# otherwise, the script exits without failing.
+
+# Retrieve the current branch name.
+branch_name=$(git rev-parse --abbrev-ref HEAD)
+
+# We expect the following syntax (case insensitive): wt-<digits>.
+regex="^(wt|WT|wT|Wt)-[0-9]+(-[a-zA-Z0-9-])?"
```

Comment:
Nitpick: As discussed this works correctly, but I'd prefer `...0-9-]+)?` for a full match on the branch name

Reply:
Fixed in 577d92cc1d2919e6bba9b0d9c04015eb9ccbc31e